01week/datatypes.js

```diff
@@ -0,0 +1,76 @@
+ 'use strict';
+
+<!-- #1 - Display the Date -->
+
+function ReturnDate() {
+const date = new Date;
+return date;
+}
+console.log (ReturnDate());
+
+<!-- #2 convert a number to a string -->
+
+function numToString () {
+const num= 11;
+const str= num.toString();
+return str;
+};
+
+console.log (numToString());
+
+<!-- #3 convert a string to a number -->
+
+function stringToNum () {
+const x= Number("1000");
+return x;
+}
+
+console.log (stringToNum());
+
+<!-- #4 different data types -->
+
+function printoutDataType (argument) {
+return typeof argument;
+}
+
+console.log(printoutDataType(1<9));
+
+<!-- #5 adds 2 numbers together -->
+
+function addNum () {
+const x = 50;
+const y = 100;
+const z = x + y;
+return z;
+}
+
+console.log (addNum());
+
+<!-- #6 runs only when 2 things are true -->
+
+const b= ((4 > 2) && (10 < 15));
```

Comment:
This shouldn't be happening in a variable. Program =  function. Also your logic returns true if 4 is greater that 2 and if 10 is less than 15. The question asks if two things (read parameters) both evaluate to true. `if (a&b)`. Remember, wrap in a function